src/ripple/app/ledger/Ledger.cpp

```diff
@@ -1173,13 +1173,13 @@ loadLedgerHelper(std::string const& sqlSuffix, Application& app, bool acquire)
 
     uint256 prevHash{}, accountHash{}, transHash{};
     if (sLedgerHash)
-        ledgerHash.SetHexExact(*sLedgerHash);
+        (void)ledgerHash.parseHex(*sLedgerHash);
     if (sPrevHash)
-        prevHash.SetHexExact(*sPrevHash);
+        (void)prevHash.parseHex(*sPrevHash);
     if (sAccountHash)
-        accountHash.SetHexExact(*sAccountHash);
+        (void)accountHash.parseHex(*sAccountHash);
     if (sTransHash)
-        transHash.SetHexExact(*sTransHash);
+        (void)transHash.parseHex(*sTransHash);
```

Comment:
Given how often we ignore the return value of `parseHex` (here and in many other files as well), what do you think about losing the `[[no discard]]` attribute on this function?